docs/index.html

```diff
@@ -1,3 +1,5 @@
+--- 
+---
```

Comment:
What's this part?

Reply:
This is Front Matter and is needed in order for Jekyll to interpret Liquid templating engine, it gives us access to to global and page variables such as site and page

Reply:
No, this creates `---` at the start of the rendered document. At least when I locally render it.